virttest/utils_test/libvirt_device_utils.py

```diff
@@ -17,9 +17,9 @@ def create_fs_xml(fsdev_dict):
     target = fsdev_dict.get("target", {'dir': 'fs_mount'})
     binary_dict = fsdev_dict.get('binary', {})
 
-    not os.path.isdir(source['dir']) and os.mkdir(source['dir'])
```

Comment:
why remove this line here? no need to mkdir?

Reply:
The dir is created in tp-libvirt now.